### DISCUSSION TITLE
请教码表输入法 Ctrl+8 自组词功能的用法

### DISCUSSION THREAD

**Discussion by UM-Li** [fcitx/fcitx5#969] [Q&A] [answered]
你好，我看文档里提到码表输入法下按 Ctrl+8 可以用刚输入的内容造词，请问这个功能具体怎样使用？

比如我想在郑码添加词语 "先 `mb.rd` 占 `id.j` → 先占 `mr.ij`"，我在文本框输入 “先占” 然后按 Ctrl+8，弹出这个对话框，下一步怎么做才能添加这个词？我试着按了字母数字、空格和回车键，似乎没有反应。

![Screenshot_20240210_102333](https://github.com/fcitx/fcitx5/assets/6215580/eaee8089-d05d-4f21-bdc4-5cc0a04dc829)

**Comment by 20NE** [comment]
同问，楼主解决了吗？

**Reply by UM-Li** [reply]
我遇到这个问题是因为当时郑码码表缺失构词码，官方仓库的码表已经补充 https://github.com/fcitx/fcitx5-table-extra/commit/75c3605dff279c9fedb49ea7f35a9054f451d594

**Comment by wengxt** [comment, marked as answer]
![图片](https://github.com/user-attachments/assets/14b6899d-3275-4153-8def-4c99e62d0ddf)
现在最新的版本应该没有问题了吧
